### ISSUE TITLE
Feature:  Support Manifest Labels

### ISSUE THREAD

**Issue by rawkode** [comtrya/comtrya#236]
**Description**

Allows manifests to be labeled in such a way that `comtrya -d apply -l setup` could be run and only a subset of the entire manifest directory is applied.

```yaml
labels:
  - setup
  - devtool

actions:
  - ...
```

Perhaps labels should be KV?

```yaml
labels:
  - phase=setup
  - target=devtool

actions:
  - ...
```

**Comment by rawkode** [comment]
cc @comtrya/maintainers

**Comment by icepuma** [comment]
Should this be in 1.0.0?

**Comment by icepuma** [comment]
Hmm. We could easily create a `labels` context and use the "where" - block ^^

**Comment by rawkode** [comment]
Not needed for 1.0 and is backwards compatible - so not too fussed. I may implement this prior to scratch my own itch though.

**Comment by rawkode** [comment]
This was implemented by #294
